src/popup/utils/index.js

```diff
@@ -9,33 +10,29 @@ export const formatDate = time =>
     hourCycle: 'h23',
   });
 
-export const mockLogin = async (options = {}) => {
-  await browser.storage.local.clear();
+export const getLoginState = async ({ backedUpSeed, balance, name, pendingTransaction }) => {
   const { mnemonic, publicKey } = testAccount;
-  const seed = mnemonicToSeed(mnemonic).toString('hex');
-  const keypair = {
+  const account = {
     publicKey,
-    privateKey: seed,
+    privateKey: mnemonicToSeed(mnemonic).toString('hex'),
+  };
+  return {
+    ...(await runMigrations()),
+    account,
+    subaccounts: [
+      {
+        name: 'Main Account',
+        publicKey: account.publicKey,
+        balance: 10,
```

Comment:
What does `balance 10` mean?

Reply:
I have just copied it from the previous version. It should be a balance of a specific subaccount, but I didn't check that. Anyway I'm proposing to drop subaccounts in #463